### PR TITLE
Translate 4-1 assignment into Simplified Chinese.

### PR DESCRIPTION
**Pull request by michael1011101**

### CODE REVIEW

File: 4-Classification/1-Introduction/translations/assignment.zh-cn.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# ̽����෽��
+
+## ˵��
+
+��[Scikit-learn](https://scikit-learn.org/stable/supervised_learning.html)�ĵ��У�����Է��ִ������ݷ��෽����ͨ���Ķ���Щ�ĵ������ǿ�����һ��Ѱ��С��Ϸ��Ŀ����Ѱ�ҷ��෽������������ĵ���ƥ�䱾�γ��е�һ����ݼ������������ʲ��ҷ���һ����෽���ļ���ʵ�֡��������ӱ�������doc�������Ȼ���¼��ݼ����������㷨���ʹ�á�
```
Comment (nikan1996) [inline 4-Classification/1-Introduction/translations/assignment.zh-cn.md:5]: Here are my suggested changes.

```diff
- 在[Scikit-learn](https://scikit-learn.org/stable/supervised_learning.html)文档中，你可以发现大量的数据分类方法。通过阅读这些文档，我们可以玩一个寻宝小游戏：目标是寻找分类方法，你可以在文档中匹配本课程中的一个数据集，解答你的疑问并且发现一个分类方法的技术实现。创建电子表格或者用doc创建表格，然后记录数据集如何与分类算法配合使用。

+ 在[Scikit-learn](https://scikit-learn.org/stable/supervised_learning.html)文档中，你可以发现有大量的数据分类方法。通过阅读这些文档，玩一个寻宝小游戏：你的目标是寻找一些分类方法，匹配一个本课程中的数据集并且提出你对这个数据集的问题，选择一个具体的分类技术。
+ 创建电子表格或者用.doc文件里面的表格，然后解释数据集要如何结合分类算法。

```

File: 4-Classification/1-Introduction/translations/assignment.zh-cn.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+
+## ���б�׼
+
+| ��׼ | ����                                                                                                                           | �й��о�                                                                                                                            | ����Ŭ�                                                                                                                                              |
+| -------- | ----------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|          | ���ĵ��У�����5�ַ����㷨�ͼ���ʵ�֡�����������ȫ������ϸ�� | ���ĵ��У�����3�ַ����㷨�ͼ���ʵ�֡�����������ȫ������ϸ��  | ���ĵ��У������3�ַ����㷨�ͼ���ʵ�֡��������ݼ�û�кܺõĽ��ͣ�Ҳû����ϸ˵�� |
```
Comment (nikan1996) [inline 4-Classification/1-Introduction/translations/assignment.zh-cn.md:12]: Here are my suggested changes.

```diff
- | | 在文档中，概述5种分类算法和技术实现。概述内容做到全面且详细。 | 在文档中，概述3种分类算法和技术实现。概述内容做到全面且详细。 | 在文档中，概述不到3种分类算法和技术实现。概述内容既没有很好的解释，也没有详细说明。 |
- 

+ | | 提供一份文档，概述5种分类技术相关的算法。概述内容做到全面且详细。 | 在文档中，概述3种分类技术相关的算法。概述内容做到全面且详细。 | 在文档中，概述不到3种分类技术相关的算法。概述内容既没有很好的解释，也不够详细。 |
```
Algorithms alongside a classification technique. 
直译是 和分类技术一起的算法
我这里翻译成 分类技术相关的算法，即这个算法可以用作分类，当然这个算法也可以用于回归等问题。

**Comment by jlooper** [comment]
hi @michael1011101  - please make the suggested edits and I'll approve, thanks!

**Review by lucalves — APPROVED** [review]

**Comment by jlooper** [comment]
@michael1011101 could you resolve the requested changes please?

**Comment by jlooper** [comment]
closing for staleness

**Pull request closed without merging**